### DISCUSSION TITLE
Options to filter which events are visible in different views

### DISCUSSION THREAD

**Discussion by ascsanto** [davish/obsidian-full-calendar#388] [Feature Requests]
I feel like this is a more particular usecase with respect to the previous requests, and many people wouldn't need this at all, so consider this request after the other ones.
My problem is that, since i have very differentiated days, i tend to use this plugin by putting a lot of events in each day, and this plugin is great for this, the point is that in the month view i would like to see only the more relevant stuff.

My idea is having an option to show only the "all day events" (or multiple day events), which are already shown in a box, to differentiate them at first sight, and this is brilliant and very useful, but i have so many events that i can't see the full month at a single glance, i just see a couple of weeks at time.

Or if you want to advance this further, some ways to see only the relevant stuff could be:
- An option to choose to show only events that are long at least a selectable amount of hours, in order to show only the main activities of the day
- An option to exclude individual calendars from the month view (i.e. i have a "miscellaneous" calendar of events that clutter the month view, and i dont really need to see them there), or contrariwise include only individual calendars (i.e. "importat" calendar)

**Comment by davish** [comment]
Thanks for the suggestion! For my own organization I'll link this to issue #48 and issue #86, since I feel they are both related to the two prongs of the request.
